src/Controller/ReservationController.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Reservation;
+use App\Form\ReservationType;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ReservationController extends AbstractController
+{
+    /**
+     * @Route("/reservation", name="reservation")
+     */
+    public function index(Request $request)
+    {
+        $dateFrom = new \DateTime('2018-11-27');
```

Comment:
kam sitie visi kintamieji sukurti?

Reply:
šie kintamieji į formą ateis iš frontend - priklausomai ką paspaus useris. Tam kad dabar veiktų mano  kodas, hardkodinau šiuos kintamuosius. Testavau kaip įrašinėja į db. Kai sujungsime su frontendu, šiuos kintamieji reikšmes gaus per post metodą.

Reply:
tai forma kaip suprantu yra? userio prisijungimas yra? kodel negalima is rezervacijos formos paimti duomenis ir sudeti i db?